### PR TITLE
feat(seo): added op-image

### PR DESCRIPTION
**Pull request by JARofStuff**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- Added op:image
- Set up HeadSeo component to work on both layouts (with and without Frontmatter)
- Set 'site' in astro.config.mjs to be current working url (https://beta.create.t3.gg)
- Removed astro-seo from LandingPage layout
- Removed unnecessary props from landing page (they were the same as the defaults set in config.ts)

### CODE REVIEW

File: .changeset/spicy-carpets-happen.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
+---
+
+Added op:image, set up headSeo for correct usage. Set config site to 'beta.create.t3.gg'
```
Comment (juliusmarminge) [inline .changeset/spicy-carpets-happen.md:5]: Not needed for docs changes.

Changesets only for the CLI

File: www/src/layouts/landingPage.astro [hunk 2]
```diff
@@ -21,7 +17,8 @@ const currentPage = Astro.url.pathname;
 <html dir="ltr" lang="en-us" class="initial">
   <head>
     <HeadCommon />
-    <SEO title={title} description={description} />
```
Comment (juliusmarminge) [inline www/src/layouts/landingPage.astro:24]: Can remove it as dependency too

File: .changeset/spicy-carpets-happen.md (hunk 1 above)
Comment (JARofStuff) [reply .changeset/spicy-carpets-happen.md]: Ok, removed the file.

File: www/src/layouts/landingPage.astro (hunk 2 above)
Comment (JARofStuff) [reply www/src/layouts/landingPage.astro:24]: Removed

**Review by juliusmarminge — APPROVED** [review]
Great

**Pull request merged**